dataprofiler/profilers/profiler_options.py

```diff
@@ -1228,15 +1229,17 @@ def __init__(
         # Non-Option variables
         self.null_values = null_values
         self.column_null_values = column_null_values
+        self.hll_row_hashing = hll_row_hashing
```

Comment:
this needs to be under row_statistics since that's the hierarchy in the code. We will have to make a class for row statistics instead of using booleanoptions which will be similar to correlations like on https://github.com/capitalone/DataProfiler/pull/841/files#diff-00da4b3383cf04d1ea70e7d179d78433f0bfde3ac6909c791eb92363c5a34f3bR1225